Client/Frontend/Settings/SettingsContentViewController.swift

```diff
@@ -107,7 +107,7 @@ class SettingsContentViewController: UIViewController, WKNavigationDelegate {
     }
 
     func makeWebView() -> WKWebView {
-        let config = WKWebViewConfiguration()
+        let config = TabManager.makeWebViewConfig(isPrivate: true, blockPopups: true)
```

Comment:
It would be nice if this method wasn't hanging off of `TabManager`, but its fine for now.